services/lgtm/lgtm-base.js

```diff
@@ -25,8 +31,13 @@ module.exports = class LgtmBaseService extends BaseJsonService {
     return { label: 'lgtm' }
   }
 
-  async fetch({ user, repo }) {
-    const url = `https://lgtm.com/api/v0.1/project/g/${user}/${repo}/details`
+  static get pattern() {
+    return `:host(${Object.keys(hostMappings).join('|')})/:user/:repo`
+  }
+
+  async fetch({ host, user, repo }) {
+    const mappedHost = host in hostMappings ? hostMappings[host] : host
```

Comment:
This conditional is no longer necessary given the pattern restriction (will always be one of the keys in `hostMappings`

Could either do
```suggestion
    const mappedHost = hostMappings[host]
```

Or delete that line and inline it in the `url` template  string